package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "testswitch-frontend-admin",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/techswitch-learners/testswitch-frontend-admin.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/techswitch-learners/testswitch-frontend-admin/issues"
+  },
+  "homepage": "https://github.com/techswitch-learners/testswitch-frontend-admin#readme",
+  "dependencies": {
+    "@zeit/next-sass": "^1.0.1",
+    "next": "^9.3.4",
+    "node-sass": "^4.13.1",
```

Comment:
`@zeit/next-sass` and `node-sass` can both be in the dev-dependencies section rather than the dependencies one.

They are both needed during the `build` step but not during the `run` step.
So we can keep our app nice and small by moving them to dev-dependencies, where they won't be added to the final application.